### DISCUSSION TITLE
Change cursor size

### DISCUSSION THREAD

**Discussion by crydotsnake** [i3/i3#4507] [Q&A] [answered]
Hello i3 Community,

I use Debian 11 with i3. And i have a problem, my mouse cursor is a little bit too big. I try creating a `.Xresources` file with the following content: `Xcursor.size: 10`. But that doesent work.. but when i use `xrdb -query` i can see the content of my `.Xresources` file. Is there something that i might do wrong?

**Comment by stapelberg** [comment, marked as answer]
Rebooting fixed this as per the reporter. I think an Xorg restart is required.

**Reply by crydotsnake** [reply]
How can i restart xorg

**Reply by stapelberg** [reply]
I think the easiest way is by restarting your display manager. Depending on your Linux distribution and setup this might be `systemctl restart gdm.service` or `systemctl restart lightdm.service` or similar.

**Reply by ghost** [reply]
Another solution; `sudo pkill X`
